pkg/configurator/crd_client.go

```diff
@@ -1,6 +1,7 @@
 package configurator
```

Comment:
Rename file to `client.go`.

Reply:
Prefer to do it in a separate PR. So that in the code review here, we can look at a complete client implementation.

Reply:
Changed mind. Make sense to use `client.go` directly